### DISCUSSION TITLE
Merging "Designs" into the book

### DISCUSSION THREAD

**Discussion by yaleman** [kanidm/kanidm#615] [General]
Currently all the design docs are a separate part of the codebase to the book and opaque to users, would it be useful to have a sub-section of the book to store all the design decisions/notes? As usual I'm happy to do the work to move and merge.

**Comment by Firstyear** [comment]
I don't think so ... the designs can often be really rough, and are detailing a lot of the inner works of the system, and in some cases (sadly) are a little out of date.

So I'd say to be "in the book" it would be in a developers section, and even then, some of the content could likely be moved into the api docs as section documentation I think.
